src/java/org/apache/cassandra/db/compaction/CompactionTask.java

```diff
@@ -66,6 +66,9 @@ public class CompactionTask extends AbstractCompactionTask
 {
     protected static final Logger logger = LoggerFactory.getLogger(CompactionTask.class);
 
+    private static final boolean COMPACTION_HISTORY_ENABLED = 
+        Boolean.parseBoolean(System.getProperty("cassandra.compaction_history_enabled", "true"));
```

Comment:
I am wonder why don't you use `CassandraRelevantProperties`? DSE doesn't have it, thus direct port will not use it.

Reply:
Yes, it was a direct port so I didn't think of it - I can change it to use `CassandraRelevantProperties`

Do you think it's worthwhile to also change the below `CURSORS_ENABLED` to do the same?

Reply:
Yes, it's good to fix at the same time. Also result of port from DSE (STAR-782)

Reply:
> Also result of port from DSE (STAR-782)
Sorry, I am not sure I follow - can you elaborate?

Reply:
I mean that line with `CURSORS_ENABLED` with added during another port from DSE. So `also port` and thus missed to use Cassandra way.

Reply:
Ok, got it - lol, I wasn't parsing the line well.

Reply:
Sorry, I made some typos above. I fixed now. I understand why it's not easy to understand me